image_classification/mobilenet_uint8_nhwc.js

```diff
@@ -0,0 +1,484 @@
+'use strict';
+
+import {buildConstantByNpy, computePadding2DForAutoPad} from '../common/utils.js';
+
+/* eslint max-len: ["error", {"code": 120}] */
+
+// MobileNet V2 model with 'nhwc' input layout
+export class MobileNetV2Uint8Nhwc {
+  constructor() {
+    this.context_ = null;
+    this.builder_ = null;
+    this.graph_ = null;
+    this.inputTensor_ = null;
+    this.outputTensor_ = null;
+    this.weightsUrl_ = '../test-data/models/mobilenetv2_nhwc/weights_uint8/';
```

Comment:
You need to submit the weight files to [test-data](https://github.com/webmachinelearning/test-data) at first.